### DISCUSSION TITLE
How to use Realistic Vision V6.0 B1 refiner with an already existing image file?

### DISCUSSION THREAD

**Discussion by larrybird1** [lllyasviel/Fooocus#1950] [Q&A]
Just this, i know it works for generating some new image, but How to use Realistic Vision V6.0 B1 refiner with an already existing image file?

**Comment by mashb1t** [comment]
Do you still have/know the config used to create the image? If not, you can try to use vary, guess the prompt and try to achieve it like this.
This is a perfect example if metadata being useful in images, which isn't possible in Fooocus until https://github.com/lllyasviel/Fooocus/pull/1940 is merged.

**Comment by toutjavascript** [comment]
You should search for your log.html file associated to this image and copy the metadata to focus UI
Use my Fooocus Log Viewer to do that work for you
https://github.com/toutjavascript/Fooocus-Log-Viewer

You find your image, you click on copy the metadata to clipboard and paste to UI prompt textarea

**Comment by docppp** [comment]
I was able to get decent results without any prompt. If image generated by juggernaut is good, but lack a bit of realism, I just use Vary (subtle) option with this exact image and add RealVis as a refiner.
